Sources/AppStoreConnectCLI/Commands/TestFlight/BetaGroups/Sync/PullBetaGroupsCommand.swift

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+import FileSystem
+
+struct PullBetaGroupsCommand: CommonParsableCommand {
+
+    static var configuration = CommandConfiguration(
+        commandName: "pull",
+        abstract: "Pull down server beta groups, refresh local beta group config files"
+    )
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Option(
+        default: "./config/betagroups",
+        help: "Path to the Folder containing the information about beta groups. (default: './config/betagroups')"
```

Comment:
The default will be printed automatically.
```suggestion
        help: "Path to the Folder containing the information about beta groups."
```